docs/overview.rst

```diff
@@ -87,8 +87,8 @@ the following operations are performed:
 1. find the backends where the database is stored
 2. find the latest version of a database (optional)
 3. calculate :ref:`database dependencies <database-dependencies>`
-4. download archive files from the selected backend
-5. unpack the archive files
+4. download (archive) files from the selected backend
```

Comment:
I wonder whether this is the right location for showing how (media) deps are obtained? Probably the documentation mentioned in 3. would be the better location as this would make things too complicated in this enumeration.  When a media file for a specific flavor is already in the cache it will be copied to the the requested flavor's media directory location, downloading and unpacking will not be necessary. At least as far as my understanding goes ...